index.html

```diff
@@ -165,11 +165,11 @@ <h2>About Myself</h2>
                         <a href="#"><img src="./image/union.svg" alt="Union" /></a> 
                     </div>
                       <ul class="language-list">
-                        <li><a href="#" class="language-icon"><img src="./image/js.png" alt="" /></a></li>
-                        <li><a href="#" class="language-icon"><img src="./image/html.png" alt="" /></a></li>
-                        <li><a href="#" class="language-icon"><img src="./image/css.png" alt="CSS" /></a></li>
-                        <li><a href="#" class="framework-icon"><img src="./image/frameworks.png" alt="frameworks"/></a></li>
-                        <li><a href="#" class="framework-icon"><img src="./image/skills.png" alt="Skills" /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/js.png" alt="Javascript Icon" /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/html.png" alt="Html Icon " /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/css.png" alt="Css Icon" /></a></li>
+                        <li><a href="#" class="framework-icon"><img src="./image/frameworks.png" alt="Frameworks Icon"/></a></li>

```

Comment:
- [ ] [OPTIONAL] Kindly make sure your `input` and `textarea` tags have `label` this helps users that rely on screen readers to know what should be done and what that `input` field represents

![image](https://user-images.githubusercontent.com/74814780/179850417-945d9382-07fb-4ade-8786-6358deb22ce2.png)